src/App.css

```diff
@@ -1,39 +1,74 @@
-.App {
-  text-align: center;
-}
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+html {
+  padding: 0;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.logo-link {
+  font-size: 1.875rem;
+  background-color: rgba(1, 1, 1, 0);
+  border: none;
+  margin-right: 3vw;
+  color: #0290ff;
+  font-family: 'Montserrat', sans-serif;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.navbar {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
+  background-color: #fff;
+  border-bottom-color: #e8e8e8;
+  border-bottom-width: 2px;
+  border-bottom-style: solid;
+  height: 5.875rem;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  width: 99vw;
 }
 
-.App-link {
-  color: #61dafb;
+.links {
+  display: flex;
+  width: 12vw;
+  justify-content: space-between;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 400;
+}
+
+.content {
+  background-color: #fafafa;
+  margin: 0;
+}
+
+.nav-item {
+  font-size: 0.813rem;
+  text-decoration: none;
+  color: #e8e8e8;
+  height: 2rem;
+  text-align: center;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+.nav-item:hover {
+  color: black;
+}
+
+.user {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 1.2rem;
+  width: 1.2rem;
+  justify-self: end;
+  padding: 10px;
+  border-color: #e8e8e8;
+  border-width: 1px;
+  border-style: solid;
+  border-radius: 50%;
+  margin-left: 58vw;
+}
+
+.icon {
+  fill: #0290ff;
+}
 
-  to {
-    transform: rotate(360deg);
-  }
+.list {
+  width: 50vw;
```

Comment:
- [ ] <b>REQUIRED:</b> There are still few things to fix to make the app look more like the provided mockup. Kindly fix them.

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/45217364/130871998-88863e2f-c823-4b7b-a5c4-20162973ff21.png) | ![scrnli_10_1_2021_6-03-31 PM](https://user-images.githubusercontent.com/45217364/135652203-33603080-a3fd-4702-8d2a-0ebccb8f4abd.png) |
  - In the toolbar, the right and left margins must be exactly the same
  - when the title is long, It shouldn't push other elements out of bound.
  - The height of form elements is way too big compared to the height of those elements in the mockup
  - There must be a line separating book list and form